core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1065,7 +1082,7 @@ class DAGScheduler(
         // the fetch failure has already been handled by the scheduler.
         if (runningStages.contains(failedStage)) {
           logInfo(s"Marking $failedStage (${failedStage.name}) as failed " +
-            s"due to a fetch failure from $mapStage (${mapStage.name})")
+              s"due to a fetch failure from $mapStage (${mapStage.name})")
```

Comment:
The spacing you changed here and above seems inconsistent with the rest of Spark, where we wrap lines with 2-space indentation.  What was the reason for changing this?